### ISSUE TITLE
[feature] Support for STM32G491 and STM32H7A3/H7B3

### ISSUE THREAD

**Issue by jjamesson** [stlink-org/stlink#1164]
Dear STLINK maintainers,

We are considering using STM32G491, and STM32H7A3/STM32H7B3 in our designs. 

As of now, it seems that these boards are not supported by ST tools yet. Could we expect support from your toolset for these chips in the near future? It would be very helpful.

With best regards!

**Comment by Ant-ON** [comment]
@jjamesson I added support for these chips earlier (https://github.com/stlink-org/stlink/commit/9eb81e1e0adc6a5bf7a1c68cbbaf3fd35ec85be0 and https://github.com/stlink-org/stlink/commit/54000d67acefd86df4bdb0594dee940dd04763fa already in the `develop` branch). I had nothing to test these chips on, but everything should work. Support for the H7 and G4 series has been added previously and has been tested.
